### PR TITLE
Letter-spacing

### LINKED ISSUE #1600: Fix letter-spacing for certain type elements
- Mobile H1 is now 3%
- Desktop H2 is now 3%
- Mobile H2 is now 3% (Modal titles)
- Mobile H3 is now 3% , modal subtitles

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #1600 

Status: Rebased and Ready for Review

## What this PR does
- Remove the old letter-spacing variable, and add letter-spacing percentage variables
- Use CSS calc to calculate letter-spacing: the font-size in rem * the letter-spacing percentage
- Use the type set by the most basic element (like `h1`, `h3`) - prefer to not add class or component-level overrides when the type is using the same styles as the already-styled basic elements.

## Testing

The browser developer tools shows letter-spacing in pixels. Make sure your Figma is in Dev Mode, with display set to Pixels, so you can compare the Figma's pixels to the browser's pixels.

### H1

**Desktop**|**Mobile**
:-----:|:-----:
 image| image
1.75px | 0.72px

#### A regular H2 where the size/letter-spacing don't change responsively, `h2, .h2`
**Desktop**|**Mobile**
:-----:|:-----: 
image|image
0.72px | 0.72px 

#### H2 that down-sizes to a P on Mobile, `.h2-sm-p`
**Desktop**|**Mobile**
:-----:|:-----:
 image|image 
0.72px | 0.9px

#### H2 that downsizes to font-size 20px on Mobile, `.h2-sm`
**Desktop**|**Mobile**
:-----:|:-----:
image|image
0.72px| 1px

### H4 
**Desktop**|**Mobile**
:-----:|:-----:
image|image
1px | 0.54px

### Body, Paragraph
**Desktop**|**Mobile**
:-----:|:-----:
image|image
0.9px | 0.9px

### Buttons
**Desktop**|**Mobile**
:-----:|:-----:
image|image
0.4px | 0.4px

### Previous Page
**Desktop**|**Mobile**
:-----:|:-----:
image|image
0.7px | 0.7px

### CODE REVIEW

**Comment by thekaveman** [comment]
This is going to cause some pretty significant conflicts with the work in #1626.

Since this is pretty straightforward class changes (and I'm headed out Wed-Fri) can this wait until after #1626 is merged?

**Comment by machikoyasuda** [comment]
This PR is code complete, but will wait on #1626 for any potential merge conflicts. Will be reviewed _after_ #1626.

**Comment by machikoyasuda** [comment]
Rebased latest dev

**Comment by machikoyasuda** [comment]
@thekaveman @angela-tran this PR is rebased with the latest `dev` and is now ready for review

**Comment by machikoyasuda** [comment]
@thekaveman @angela-tran Wait sorry no it's not!!! One sec

**Comment by machikoyasuda** [comment]
@thekaveman @angela-tran _Nowwwww_ ready for review

**Review by angela-tran — APPROVED** [review]

**Pull request merged**
